product_harmonized_system/models/hs_code.py

```diff
@@ -0,0 +1,74 @@
+# © 2011-2016 Akretion (http://www.akretion.com)
+# © 2009-2016 Noviat (http://www.noviat.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# @author Luc de Meyer <info@noviat.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class HSCode(models.Model):
+    _name = "hs.code"
+    _description = "H.S. Code"
+    _order = "local_code"
+    _rec_name = "display_name"
+
+    hs_code = fields.Char(
+        string='H.S. Code', compute='_compute_hs_code', readonly=True,
+        help="Harmonized System code (6 digits). Full list is "
+        "available from the World Customs Organisation, see "
+        "http://www.wcoomd.org")
+    description = fields.Char(
+        'Description', translate=True,
+        help="Short text description of the H.S. category")
+    display_name = fields.Char(
+        compute='_compute_display_name_field', string="Display Name",
```

Comment:
No need of `string="Display Name"`

Reply:
Code review & tested. 
Remarks of @astirpe are valuable hence would be nice to include but imho not blocking for merging this PR.

Reply:
@luc-demeyer ... however it's stated in the OCA Guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields